flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/StreamingFileSink.java

```diff
@@ -275,22 +282,30 @@ private StreamingFileSink(
 			this.bucketer = Preconditions.checkNotNull(bucketer);
 		}
 
+		BulkFormatBuilder(
```

Comment:
Same as above, this constructor can be `private` and call this constructor from the previous one.